test/distributed/_composable/fsdp/test_fully_shard_clip_grad_norm_.py

```diff
@@ -83,7 +74,12 @@ def _test_clip_grad_norm_1d(
                     foreach=True,
                 )
             self.assertEqual(ref_total_norm, total_norm)
-            self.assertEqual(comm_mode.get_total_counts(), 1)  # one all-reduce
+            # Expect one all-reduce per mesh dim for partial -> replicate
```

Comment:
@wanchaol JFYI, I see 1 all-reduce per mesh dim.

Reply:
Yeah this make sense and looks like FS also have 2 allreduces